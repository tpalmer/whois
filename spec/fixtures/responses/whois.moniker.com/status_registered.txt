
Moniker Whois Server Version 2.0

The Data in Moniker's WHOIS database
is provided for information purposes only, and is
designed to assist persons in obtaining information
related to domain name registration records.
Moniker does not guarantee its accuracy.
By submitting a WHOIS query, you agree that you
will use this Data only for lawful purposes and
that, under no circumstances will you use this Data
to: (1) allow, enable, or otherwise support the
transmission of mass unsolicited, commercial
advertising or solicitations via e-mail (spam),
telephone, or facsimile; or
(2) enable high volume, automated, electronic
processes that apply to Moniker (or its
systems). Moniker reserves the right
to modify these terms at any time.  By submitting
this query, you agree to abide by this policy.

The compilation, repackaging, dissemination or
other use of this Data is expressly
prohibited without the prior written consent
of Moniker.

Domain Name: MONIKER.COM
Registrar: MONIKER

Registrant [3793977]:
        Domain Manager domains@moniker.com
        Moniker Online Services, LLC
        20 SW 27th Ave.
        Suite 201
        Pompano Beach
        FL
        33069
        US


Administrative Contact [3793977]:
        Domain Manager domains@moniker.com
        Moniker Online Services, LLC
        20 SW 27th Ave.
        Suite 201
        Pompano Beach
        FL
        33069
        US
        Phone: +1.9549848445
        Fax:   +1.9549699155


Billing Contact [3793977]:
        Domain Manager domains@moniker.com
        Moniker Online Services, LLC
        20 SW 27th Ave.
        Suite 201
        Pompano Beach
        FL
        33069
        US
        Phone: +1.9549848445
        Fax:   +1.9549699155


Technical Contact [3793977]:
        Domain Manager domains@moniker.com
        Moniker Online Services, LLC
        20 SW 27th Ave.
        Suite 201
        Pompano Beach
        FL
        33069
        US
        Phone: +1.9549848445
        Fax:   +1.9549699155


Domain servers in listed order:

        NS1.MONIKER.COM         207.189.109.115
        NS2.MONIKER.COM         63.156.206.115
        NS3.MONIKER.COM         50.57.11.89
        NS4.MONIKER.COM         50.57.11.88

        Record created on:        2004-03-18 14:05:56.793
        Database last updated on: 2012-06-07 16:37:30.3
        Domain Expires on:        2017-03-04 00:00:00.0



