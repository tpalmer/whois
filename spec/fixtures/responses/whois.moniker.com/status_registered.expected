#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2004-03-18 14:05:56 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-06-07 16:37:30 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2017-03-04 00:00:00 UTC")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "Moniker"
  should: %s.organization == "Moniker Online Services LLC"
  should: %s.url          == "http://moniker.com"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id           == "3793977"
  should: %s[0].name         == "Domain Manager"
  should: %s[0].organization == "Moniker Online Services, LLC"
  should: %s[0].address      == "20 SW 27th Ave.\nSuite 201"
  should: %s[0].city         == "Pompano Beach"
  should: %s[0].zip          == "33069"
  should: %s[0].state        == "FL"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == nil
  should: %s[0].fax          == nil
  should: %s[0].email        == "domains@moniker.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id           == "3793977"
  should: %s[0].name         == "Domain Manager"
  should: %s[0].organization == "Moniker Online Services, LLC"
  should: %s[0].address      == "20 SW 27th Ave.\nSuite 201"
  should: %s[0].city         == "Pompano Beach"
  should: %s[0].zip          == "33069"
  should: %s[0].state        == "FL"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.9549848445"
  should: %s[0].fax          == "+1.9549699155"
  should: %s[0].email        == "domains@moniker.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id           == "3793977"
  should: %s[0].name         == "Domain Manager"
  should: %s[0].organization == "Moniker Online Services, LLC"
  should: %s[0].address      == "20 SW 27th Ave.\nSuite 201"
  should: %s[0].city         == "Pompano Beach"
  should: %s[0].zip          == "33069"
  should: %s[0].state        == "FL"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.9549848445"
  should: %s[0].fax          == "+1.9549699155"
  should: %s[0].email        == "domains@moniker.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.moniker.com"
  should: %s[0].ipv4 == "207.189.109.115"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.moniker.com"
  should: %s[1].ipv4 == "63.156.206.115"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns3.moniker.com"
  should: %s[2].ipv4 == "50.57.11.89"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns4.moniker.com"
  should: %s[3].ipv4 == "50.57.11.88"
